x/campaign/keeper/campaign_auction_event_hooks.go

```diff
@@ -0,0 +1,199 @@
+package keeper
+
+import (
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	fundraisingtypes "github.com/tendermint/fundraising/x/fundraising/types"
+	"github.com/tendermint/spn/x/campaign/types"
+	profiletypes "github.com/tendermint/spn/x/profile/types"
+)
+
+// EmitCampaignAuctionCreated emits EventCampaignAuctionCreated event if an auction is created for a campaign from a coordinator
+func (k Keeper) EmitCampaignAuctionCreated(
+	ctx sdk.Context,
+	auctionId uint64,
+	auctioneer string,
+	sellingCoin sdk.Coin,
+) (bool, error) {
+	campaignID, err := types.VoucherCampaign(sellingCoin.Denom)
+	if err != nil {
+		// not a campaign auction
+		return false, nil
+	}
```

Comment:
```suggestion
	campaignID, err := types.VoucherCampaign(sellingCoin.Denom)
	if err == types.ErrNotAuctionCampaign {
		return false, nil	
	} else if err != nil {	
		return false, err
	}
```

Reply:
I think this is fine `VoucherCampaign ` simply returns an error in general if the coin is not a voucher